src/main/resources/db/MTB dml.sql

```diff
@@ -10,3 +10,32 @@ values ('BIKE', 'bike'),
 insert into gender (code)
 values ('MALE'),
        ('FEMALE');
+
+insert into size (code)
+values  ('XXS'),
+        ('XS'),
+        ('S'),
+        ('M'),
+        ('L'),
+        ('XL'),
+        ('XXL');
+
+insert into color (code)
+values  ('BLACK'),
+        ('WHITE'),
+        ('YELLOW'),
+        ('RED'),
+        ('PURPLE'),
+        ('PINK'),
+        ('GREEN'),
+        ('GRAY'),
+        ('BLUE');
+
+insert into brand (code)
```

Comment:
brand table does not have a code. Change to name here

Reply:
will wait for changes

Reply:
ok, changed